api/spec/services/expense_sheet_generator_spec.rb

```diff
@@ -127,4 +143,17 @@
       end
     end
   end
+
+  describe '#create_missing_expense_sheets' do
+    before do
+      expense_sheet_generator.create_expense_sheets
+      service.update ending: service.ending + 28.days
+      allow(expense_sheet_generator).to receive(:create_expense_sheets)
+    end
+
+    it 'calls create_expense_sheets with the correct arguments' do
+      expense_sheet_generator.create_missing_expense_sheets
```

Comment:
```suggestion
      expect { expense_sheet_generator.create_missing_expense_sheets }.to change(ExpenseSheet, :count).by(1)
```

Reply:
This is already spec'd in ExpenseSheetGenerator#create_expense_sheets